.circleci/config.yml

```diff
@@ -0,0 +1,126 @@
+#
+# Copyright (c) 2021, Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+version: 2.1
+
+executors:
+  daml-docker-executor:
+    docker:
+      - image: cimg/openjdk:11.0
+
+commands:
+  install_sdk:
+    description: "Install Daml SDK"
+    steps:
+      - run:
+          name: Install Daml SDK
+          command: |
+            curl -sSL https://get.daml.com/ | sh /dev/stdin ${SDK_VERSION}
+            # idea from https://circleci.com/docs/2.0/env-vars/
+            >> $BASH_ENV echo 'export PATH="$HOME/.daml/bin:$PATH"'
+  install_yq:
```

Comment:
Does there not exist an image that already has yq installed? For instance, you said it's used in the marketplace. What docker-image does it use?

Reply:
I unashamedly copied this installation config directly from the Marketpace - https://github.com/digital-asset/da-marketplace/blob/main/.circleci/config.yml#L34

The docker image used is specified in the config script (see https://github.com/digital-asset/daml-ctl/blob/CI_Setup/.circleci/config.yml#L11) is `cimg/openjdk:11.0` which is supplied by circleci for JVM projects. See the overview [here](https://circleci.com/docker/) and the image docs [here](https://circleci.com/developer/images/image/cimg/openjdk). We could create our own image using CircleCI's image as a base which would contain all the necessary libraries required to run our builds.

Reply:
I was hoping more that there was an existing image that had the tooling we needed. Otherwise we'll end up having to maintain a docker image, which I don't think we want to do.

Maybe worth thinking of this as part of a larger initiative in the company (many projects need things like tools for reading `yaml` files etc.)

Also worth checking whether the build system for the daml SDK has an image that might have useful stuff in it.

Reply:
> Also worth checking whether the build system for the daml SDK has an image that might have useful stuff in it.

There's no existing image that contains the tooling we need. The SDK images contain the Daml SDK but nothing for tooling (https://github.com/digital-asset/daml/blob/main/ci/docker/daml-sdk/Dockerfile). 

> Use the enterprise docker image (if we're allowed).

These images are hosted on Artifactory - downloading images from here requires an enterprise key. From the docs, CircleCI allows for pulling images from Docker or AWS ECR. 

> Maybe worth thinking of this as part of a larger initiative in the company (many projects need things like tools for reading yaml files etc.)

Personally, I think this is better per dev team. We know what tools we need and what version they are; having to coordinate across all dev teams will lead to bloat and bureaucracy in my view.